govuk-prototype-kit/templates/autocomplete.html

```diff
@@ -0,0 +1,66 @@
+{% extends "layouts/main.html" %}
+
+{% block pageTitle %}
+  Autocomplete question page template – {{ serviceName }} – GOV.UK Prototype Components
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    href: "javascript:window.history.back()"
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <form class="form" action="/url/of/next/page" method="post">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        {{ xGovukAutocomplete({
+          id: "country",
+          name: "country",
+          allowEmpty: false,
+          label: {
+            classes: "govuk-label--l",
+            isPageHeading: true,
+            text: "Pick a country"
```

Comment:
I don’t think we’d start a question with “Pick” usually? (I know this is just an example template, but still)

```suggestion
            text: "Country of birth"
```

Reply:
(I use "Pick" as a verb in questions quite frequently)